de.metas.procurement.base/src/main/sql/postgresql/ddl/functions/docs_flatrate_term_all_procurements_conditions.sql

```diff
@@ -452,11 +452,13 @@ SELECT	C_Flatrate_Term_ID,
 
 FROM C_Flatrate_Term ft
 JOIN PMM_Product p ON p.PMM_Product_ID = ft.PMM_Product_ID
+JOIN C_Flatrate_Conditions fc ON ft.C_Flatrate_Conditions_ID = fc.C_Flatrate_Conditions_ID
 WHERE ft.type_conditions='Procuremnt' AND ft.isActive='Y'
 	AND ft.startdate::date<=now()::date AND ft.enddate::date>=now()::date
 	AND ft.PMM_Product_ID IS NOT NULL AND Bill_BPartner_ID IS NOT NULL
 	AND COALESCE(ft.Bill_BPartner_ID = $1, true)
 	AND COALESCE(p.M_Product_ID = $2, true)
+	AND NOT EXISTS (SELECT r.C_Flatrate_Conditions_ID FROM C_RfQ r WHERE fc.C_Flatrate_Conditions_ID = r.C_Flatrate_Conditions_ID)
```

Comment:
i checked the datamodel a bit: there is a column `C_RfQResponseLine.C_Flatrate_Term_ID`; the term is created and referenced from there when the RfQ is closed (thx @metas-dh ).
So, what about "beeing referenced by a C_RfQResponseLine" to be the criterion to exclude a term?

Asking because I'm a bit uneasy about excluding a term because its conditions is referenced from some RfQ..you can spoil the excel just by creating a C_RfQ record and selecting a particular conditions.

Btw, please generally remember to ignore inactive records and document that are not completed or closed.